docs/BreakingChanges/141 WPF TreeViewItem must be used within a TreeView.md

```diff
@@ -0,0 +1,27 @@
+## ID: WPF TreeViewItem must be used within a TreeView
+
+### Scope
+Major
+
+### Version Introduced
+4.5
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+Before 4.5, a TreeViewItem could be used outside of a TreeView. A change in 4.5 added a check during scrolling that assumes TreeViewItems is only used in the context of 
+TreeView. An example of when this is hit is if a TreeViewItem is the root of a DataTemplate or is used without being wihtin a TreeView.
+
+Description goes here.
+
+- [ ] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+The TreeViewItem should be used from within a TreeView, so the XAML must be reworked to ensure that condition is met.
+
+### Affected APIs
+* Not detectable via API analysis
+
+[More information](LinkForMoreInformation)
```

Comment:
Should there be a link for more information? Or is this just from the template? :)